soc/ite/ec/it51xxx/soc.c

```diff
@@ -102,11 +102,8 @@ void soc_prep_hook(void)
 	struct gpio_ite_ec_regs *const gpio_regs = GPIO_ITE_EC_REGS_BASE;
 	struct gctrl_ite_ec_regs *const gctrl_regs = GCTRL_ITE_EC_REGS_BASE;
 
-	/* Scratch ROM0 is 4kb size */
-	gctrl_regs->GCTRL_SCR0SZR = IT51XXX_GCTRL_SCRSIZE_4K;
```

Comment:
This upload removes the redundant line that was mistakenly left, which has already been moved to ilm.c (line 129) for execution and is no longer needed here.